liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/ExplodedLooseWarAppTest.java

```diff
@@ -81,7 +82,8 @@ public void configureFilteredResource() throws Exception {
 		   
 		   // Verify exploded goal running and redeploy
 		   assertTrue(getLogTail(), verifyLogMessageExists("Running liberty:deploy", 2000));
-		   assertTrue(getLogTail(), verifyLogMessageExists("Running maven-war-plugin:exploded", 2000));
+		   assertTrue(getLogTail(), verifyLogMessageExists("Running                 <artifactId>maven-war-plugin</artifactId>
```

Comment:
So the syntax of the log message changed? Is that due to running with Java 17, or with a later version of Liberty? Does this change still pass with Java 8 and 11?

Reply:
This was an accidental mistake while searching/replacing text.